daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -645,12 +657,42 @@ class EngineTest
     "reinterpret to the same result" in {
 
       val reinterpretResult =
-        reinterpret(engine, Set(alice), tx.roots, tx, txMeta, let, lookupPackage, defaultContracts)
+        reinterpret(
+          engine,
+          Set(alice),
+          tx.roots,
+          tx,
+          txMeta,
+          let,
+          lookupPackage,
+          defaultContracts,
+          defaultKeys)
```

Comment:
Similar to how we drop the `byKey` flags below, would it make sense to empty the `byKeyNodes` in the metadata for this test?

Reply:
I see. Thank you.